backend/main/serializers/category_serializer.py

```diff
@@ -1,10 +1,14 @@
 from rest_framework import serializers
 from ..models.category import Category
+from ..models.lesson import Lesson
 
 class CategorySerializer(serializers.ModelSerializer):
+    lesson_id = serializers.SerializerMethodField('get_lesson_id')
```

Comment:
This can be empty since nana mn kay `get_lesson_id()` sa ubos
```suggestion
    lesson_id = serializers.SerializerMethodField()
```